relay/relay.go

```diff
@@ -498,14 +498,21 @@ func (r *Relay) reSetupMeta() error {
 		return err
 	}
 
+	latestPos, latestGTID, err := utils.GetMasterStatus(r.db, r.cfg.Flavor)
```

Comment:
should we always call `GetMasterStatus` or only when not pos/GTIDs are given?

Reply:
update in https://github.com/pingcap/dm/pull/304/commits/cd77953171a8b31e7a69f0fcb14c8fec8e0be5cd